src/foundry/client/applications/api/dialog.d.mts

```diff
@@ -386,9 +398,9 @@ declare namespace DialogV2 {
   type Content<Data extends AnyObject, BaseType extends string | HTMLDivElement = string | HTMLDivElement> = BaseType &
     Internal.FormContent<Data>;
 
-  type RenderCallback = (event: Event, dialog: HTMLDialogElement) => void;
+  type RenderCallback = (event: Event, dialog: DialogV2) => void;
```

Comment:
This is something I noticed (and fixed) on main but this should've been `DialogV2.Any` since the beginning. Could you merge in main?